client/src/pages/ElementPage.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState, useEffect } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+import { Line } from "react-chartjs-2";
+import Chart from 'chart.js/auto';
```

Comment:
I'm curious about import chart and line from two different packages. I believe one of the packages can contain both. For consistency, let's import them from one package.